lib/generators/command_service_object/install_generator.rb

```diff
@@ -0,0 +1,28 @@
+# require 'rails/generators'
+# require 'rails/generators/base'
+
+module CommandServiceObject
+  class InstallGenerator < ::Rails::Generators::Base
```

Comment:
what if we use it inside https://github.com/adham90/command_service_object/blob/feature/configuration/lib/generators/service/install/install_generator.rb so user can just run `rails g service:install` and it will create services dir

Reply:
I thought about that, but i think the common convention is using the gem name to install its configuration (devise, rollbar). It is slightly different in our case, so feel free to change it if you think its better.